templates/admin.php

```diff
@@ -11,8 +11,70 @@
  */
 
 ?>
-
-<div class="section">
-	<h2><?php p($l->t('Updater')) ?></h2>
-	<a class="button" target="_blank" href="<?php p(\OCP\Util::linkTo('updater', 'update.php')) ?>"><?php p($l->t('Open Update Center')) ?></a>
+<?php $isNewVersionAvailable = $_['isNewVersionAvailable']?>
+<div ng-app="updater" ng-init="navigation='backup'" class="updater-admin">
+	<div class="section" ng-controller="updateCtrl">
+		<h2><?php p($l->t('Updates')) ?></h2>
+		<p id="update-info" ng-show="<?php p($isNewVersionAvailable) ?>">
+			<?php print_unescaped($l->t('<strong>A new version is available: %s</strong>', array($_['version']))) ?>
+		</p>
+		<p ng-show="<?php p(!$isNewVersionAvailable) ?>">
+			<?php print_unescaped($l->t('<strong>Up to date.</strong> <em>Checked on %s</em>', array($_['checkedAt']))) ?>
+		</p>
+		<div class="upd-step-title" style="display:none;">
+			<ul class="track-progress" data-steps="3">
+				<li class="icon-breadcrumb">
+					<?php p($l->t('1. Check & Backup')) ?>
+					<span class="updater-spinner icon-loading-small"></span>
+				</li>
+				<li class="icon-breadcrumb">
+					<?php p($l->t('2. Download & Extract')) ?>
+					<span class="updater-spinner icon-loading-small"></span>
+				</li>
+				<li>
```

Comment:
Yes, because we only need the arrow between the items. Adding it here would add an arrow to the far right, which would look off.